projects/map/src/lib/helpers/server-type.helper.ts

```diff
@@ -1,15 +1,36 @@
+import { AppLayerWithServiceModel, ServiceHiDpiMode } from '@tailormap-viewer/api';
 
 export class ServerTypeHelper {
-  public static getFromUrl(url: string): undefined | 'geoserver' | 'mapserver' {
+  public static getFromUrl(url: string): ServiceHiDpiMode.GEOSERVER | ServiceHiDpiMode.MAPSERVER | undefined {
```

Comment:
Maybe its good to rename the ServiceHiDpiMode to a more generic ServerType since its now used in different contexts.

Reply:
Now that I'm thinking about it, maybe we could even add this property to the service inside the state directly when retreiving services from the backend, since this won't change and that way its already a property for a service.